tools/src/main/java/org/apache/kafka/tools/ShareConsumerPerformance.java

```diff
@@ -79,7 +79,8 @@ public static void main(String[] args) {
                 shareConsumers.forEach(shareConsumer -> shareConsumersMetrics.add(shareConsumer.metrics()));
             }
             shareConsumers.forEach(shareConsumer -> {
-                Map<TopicIdPartition, Optional<KafkaException>> val = shareConsumer.commitSync();
+                @SuppressWarnings("UnusedLocalVariable")
+                Map<TopicIdPartition, Optional<KafkaException>> ignored = shareConsumer.commitSync();
```

Comment:
@AndrewJSchofield do we need to check the result of commitSync?

Reply:
I think the local was introduced because of a false alarm in spotbugs. It claims `commitSync` is side effect free (which is not true). We need to either annotate `commitSync` with `@CanIgnoreReturnValue` or have this workaround. I'd prefer having this workaround, as this is just performance testing code and we don't want to annotate user-facing interfaces like the share consumer with spotbugs annotations.

Reply:
I completely missed that this was in `tools`. I agree that the approach here (the explicit suppression) is better

Reply:
This is OK with me in this case. It's just making sure that any in-flight acknowledgements are committed before the consumer is closed in a performance test.